### ISSUE TITLE
master branch

### ISSUE THREAD

**Issue by cvvergara** [pgRouting/docker-pgrouting#22]
@cayetanobv
Are you keeping a docker on master branch?
Rigth now it has v3.0.0-rc1 (which I will release after I fix some issues)
What about the latest tags?
like: https://github.com/pgRouting/pgrouting/releases/tag/v3.0.0-beta

**Comment by cayetanobv** [comment]
Hi @cvvergara . We maintain a docker image only for each released version and not linked to branches  (remember this issue https://github.com/pgRouting/docker-pgrouting/issues/4). So for v3.0.0 when you release the final version I will release this new docker image.

**Comment by smellman** [comment]
@cvvergara We support master branch now.

**Comment by sanak** [comment]
Yes. DockerHub tag name which ends with `-main` means the latest `main` branch image.
* https://hub.docker.com/r/pgrouting/pgrouting/tags?name=-main

I close this issue.
